library_schemas/score/prerelease/HED_score_1.0.0.mediawiki

```diff
@@ -78,49 +78,49 @@ This project was supported by the by the National Institute of Mental Health of
+    * Sleep-K-complex <nowiki> {suggestedTag=Finding-significance}[A burst of somewhat variable appearance, consisting most commonly of a high voltage negative slow wave followed by a smaller positive slow wave frequently associated with a sleep spindle. Duration greater than 0.5 s. Amplitude is generally maximal in the frontal vertex. K complexes occur during nonREM sleep, apparently spontaneously, or in response to sudden sensory / auditory stimuli, and are not specific for any individual sensory modality.]</nowiki>
+    * Sleep-saw-tooth-waves <nowiki> {suggestedTag=Finding-significance} </nowiki>
+    * Sleep-POSTS <nowiki> {suggestedTag=Finding-significance} [Positive occipital sharp transients of sleep. Sharp transient maximal over the occipital regions, positive relative to other areas, apparently occurring spontaneously during sleep. May be single or repetitive. Amplitude varies but is generally bellow 50 microV.]</nowiki>
+    * Hypnagogic-hypersynchrony {suggestedTag=Finding-significance} <nowiki>[Bursts of bilateral, synchronous delta or theta activity of large amplitude, occasionally with superimposed faster components, occurring during falling asleep or during awakening, in children.]</nowiki>
+    * Non-reactive-sleep <nowiki> {suggestedTag=Finding-significance} [EEG activity consisting of normal sleep graphoelemts, but which cannot be interrupted by external stimuli/ the patient cannot be waken.]</nowiki>
+
+
+'''Interictal-finding''' <nowiki>[EEG pattern / transient that is distinguished form the background activity, considered abnormal, but is not recorded during ictal period (seizure) or postictal period; the presence of an interictal finding does not necessarily imply that the patient has epilepsy.]</nowiki>
 
     * Epileptiform-interictal-activity
     * Abnormal-interictal-rhythmic-activity
     * Interictal-special-patterns
-        ** Interictal-periodic-discharges <nowiki>[Periodic discharges not further specified (PDs).]</nowiki>
-        ** Generalized-periodic-discharges <nowiki>[GPDs]</nowiki>
-        ** Lateralized-periodic-discharges <nowiki>[LPDs]</nowiki>
-        ** Bilateral-independent-periodic-discharges <nowiki>[BIPDs]</nowiki>
-        ** Multifocal-periodic-discharges <nowiki>[MfPDs]</nowiki>
+        ** Interictal-periodic-discharge <nowiki>[Periodic discharge not further specified (PD).]</nowiki>
+        ** Generalized-periodic-discharge <nowiki>[GPD.]</nowiki>
+        ** Lateralized-periodic-discharge <nowiki>[LPD.]</nowiki>
+        ** Bilateral-independent-periodic-discharge <nowiki>[BIPD.]</nowiki>
+        ** Multifocal-periodic-discharge <nowiki>[MfPD.]</nowiki>
         ** Interictal-extreme-delta-brush
         ** Interictal-burst-suppression <nowiki>[]</nowiki>
         ** Interictal-burst-attenuation <nowiki>[]</nowiki>
 
```

Comment:
Discharges are used in plural throughout the paper and software, checking whether it changes the meaning when changing to singular.